Code/autopkglib/MunkiSetDefaultCatalog.py

```diff
@@ -16,14 +16,16 @@
 # limitations under the License.
 """See docstring for MunkiSetDefaultCatalog class"""
 
-from autopkglib import Processor
+from __future__ import print_function
```

Comment:
This isn't necessary anymore if you're replacing print() with log().

Reply:
Ok. So just to verify. Processor imports the `log()` from `autopkglib`. `log()` uses `print()` but we don't need to import the `print()` because it was already imported in `autopkg` [itself](https://github.com/autopkg/autopkg/blob/eaf5ae258ab921ca6dec21e1eddb674e920825ea/Code/autopkg#L19). Correct?

I also removed this import from others processors. See f584460.

Reply:
I think the point is more that since you are no longer calling print(), having replaced all those calls with log(), you no longer need to from __future__ import print_function, and if you leave that in, the various linters might complain about unused imports.